assembl/static2/css/components/multiColumnAdmin.scss

```diff
@@ -0,0 +1,21 @@
+.multicolumn-admin {
+  .admin-export-section {
+    .admin-content {
+      text-align: center;
+
+      .form-group {
+        text-align: left;
+        display: inline-block;
+
+        .export-language-dropdown {
+          margin-top: 20px;
+          position: absolute;
```

Comment:
I think because there is no position on any of the parents, this position will be absolute to the viewport, which might not be what you're looking for in case of mobile. Please verify.

Reply:
Oh yea! There is no mobile admin section! Okay, then keep this. Good call.